images/gcloud-bazel/BUILD.bazel

```diff
@@ -14,3 +14,62 @@
 
 # TODO(fejta): replace Dockerfile with an equivalent bazel rule. This seems promising:
 #   https://github.com/GoogleContainerTools/base-images-docker/blob/master/package_managers/bootstrap_image.bzl
+
+load("@io_bazel_rules_docker//container:image.bzl", "container_image")
+load("@io_bazel_rules_docker//container:layer.bzl", "container_layer")
+load("@io_bazel_rules_docker//container:push.bzl", "container_push")
+load("@base_images_docker//util:run.bzl", "container_run_and_commit")
+
+# Gcloud layer. Simply includes an extractec version of the gcloud installation
+# archive. Gcloud isn't fully installed as a result of this. The final containe
+# needs to run /google-cloud-sdk/install.sh -q and source /google-cloud-sdk/path.bash.inc
+# in their bashrc to complete the gcloud installation
+container_layer(
+    name = "gcloud-layer",
+    tars = [
+        "@gcloud_archive//file",
+    ],
+)
+
+# Build an image that uses the official bazel base image and includes the gcloud
+# layer on top
+container_image(
+    name = "image_intermediate",
```

Comment:
gcloud_installer

Reply:
Are you suggesting change the comment to say "includes the gcloud_installer layer on top"?

Reply:
Never mind. I see you want to name the rule gcloud_installer